Helper/Data.php

```diff
@@ -462,7 +466,12 @@ public function getDhlApiUrl()
      */
     public function getDhlApiKey()
     {
-        return $this->getConfigValue(self::DHL_API_KEY, $this->storeManager->getStore()->getId());
+        if ($this->getConfigValue(self::CONFIG_API_MODE, $this->storeManager->getStore()->getId())
```

Comment:
its a little weird to use the getId in this way.. could the id be 1 and not be in sandbox mode?